roles/edpm_kernel/handlers/main.yml

```diff
@@ -17,3 +17,9 @@
 - name: Delete PageKSM
   ansible.builtin.command: echo 2 >/sys/kernel/mm/ksm/run
   become: true
+  changed_when: false
+
+- name: Reload sysctl
+  ansible.builtin.systemd:
+    name: systemd-sysctl.service
```

Comment:
oh your missing become:true here

Reply:
true that!

Reply:
sweet this now passes https://review.rdoproject.org/zuul/build/7482bc06971c4b4fa8d4c682cd700ab8